lib/GTK/Simple/NativeLib.pm6

```diff
@@ -92,18 +92,11 @@ sub gobject-lib is export {
 
 sub find-bundled($lib is copy) {
     # if we can't find one, assume there's a system install
-    my $base = "lib/GTK/$lib";
-    for @*INC {
-        if my @files = ($_.files($base) || $_.files("blib/$base")) {
-            my $files = @files[0]<files>;
-            my $tmp = $files{$base} || $files{"blib/$base"};
+    my $base = "blib/lib/GTK/$lib";
 
-            # copy to a temp dir
-            $tmp.IO.copy($*SPEC.tmpdir ~ '\\' ~ $lib);
+    if my $file = %?RESOURCES{$base} {
+            $file.IO.copy($*SPEC.tmpdir ~ '\\' ~ $lib);
```

Comment:
The copy is needed because windows uses the actual on-disk filename to resolve DLL dependencies, and perl6 resources get mangled sha filenames.

The temp directory is just because we aren't guaranteed to have write access to the install directory.